packages/itwin/tree-widget/src/tree-widget-react/components/trees/common/components/EmptyTree.tsx

```diff
@@ -48,15 +48,16 @@ export function TooManyInstancesFocused({ base }: FilterEmptyTreeProps) {
   return (
     <div className={"tw-filter-empty-tree-container"}>
       <Text variant={"body-sm"}>{TreeWidget.translate(`${base}.filtering.tooManyInstancesFocused`)}</Text>
-      <Anchor
-        style={{ textDecoration: "underline" }}
-        onClick={(e) => {
-          e.stopPropagation();
-          toggle();
-        }}
-      >
-        {TreeWidget.translate(`${base}.filtering.disableInstanceFocusMode`)}
-      </Anchor>
+      <Text variant={"body-sm"}>
+        <Anchor
```

Comment:
could we merge the these `Text` components into one by placing localized string and anchor under one text component?